huaweicloud/services/as/resource_huaweicloud_as_planned_task.go

```diff
@@ -0,0 +1,367 @@
+package as
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"strconv"
+	"strings"
+
+	"github.com/hashicorp/go-multierror"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/validation"
+
+	"github.com/chnsz/golangsdk"
+	"github.com/chnsz/golangsdk/openstack/autoscaling/v1/scheduledtasks"
+
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/common"
+	"github.com/huaweicloud/terraform-provider-huaweicloud/huaweicloud/config"
+)
+
+var (
+	PlannedTaskRecurrencePeriods = []string{"DAILY", "WEEKLY", "MONTHLY"}
```

Comment:
```suggestion
	plannedTaskRecurrencePeriods = []string{"DAILY", "WEEKLY", "MONTHLY"}
```
only this package used.

Reply:
fixed.